tests/test_clr.py

```diff
@@ -0,0 +1,49 @@
+import pytest
```

Comment:
pytest is rad, and these tests are just so darn concise. I do wonder if we should instead just use `unittest.TestCase`, though, to match what other Color devs expect?

(I personally think we ought just use `pytest` here, but I'm raising the question since I think others might ask it too).

Reply:
I'm happy either way. Easy to convert later if we better integrate the codebases?